homeassistant/components/blebox/button.py

```diff
@@ -0,0 +1,48 @@
+"""BleBox button entities implementation."""
+from __future__ import annotations
+
+from homeassistant.components.button import ButtonDeviceClass, ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import BleBoxEntity, create_blebox_entities
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up a BleBox button entry."""
+    create_blebox_entities(
+        hass, config_entry, async_add_entities, BleBoxButtonEntity, "buttons"
+    )
+
+
+class BleBoxButtonEntity(BleBoxEntity, ButtonEntity):
+    """Representation of BleBox buttons."""
+
+    def __init__(self, feature):
+        """Initialize a BleBox button feature."""
+        super().__init__(feature)
+        self._attr_device_class = ButtonDeviceClass.UPDATE
```

Comment:
Does the button entity perform a firmware update when pressed?

Reply:
Oh, now i see. It doesn't, it's just to run command(eg. go_up, stop) behind endpoint. So this attribute should be omitted I believe?
I'll update this in following PR.

Reply:
Yes. :+1: 

Thanks!